src/manage-fonts/index.js

```diff
@@ -181,16 +182,26 @@ function ManageFonts() {
 				{ fontToDelete?.fontFamilyIndex !== undefined &&
 				fontToDelete?.fontFaceIndex !== undefined ? (
 					<h3>
-						{ __(
-							`Are you sure you want to delete "${ fontFaceToDelete?.fontStyle } - ${ fontFaceToDelete?.fontWeight }"  variant of "${ fontFamilyToDelete?.fontFamily }" from your theme?`,
-							'create-block-theme'
+						{ sprintf(
```

Comment:
Maybe not an issue, but I find it a strange that the i18n implements its own string formatting function that mirrors the php one instead of just using ES6+ template literals.